tests/e2e/specs/shopper/shopper-wc-blocks-checkout-failures.spec.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+const { shopper, merchant, uiUnblocked } = require( '@woocommerce/e2e-utils' );
```

Comment:
can't we use regular imports here? 

```js
import { shopper, merchant, uiUnblocked } from '@woocommerce/e2e-utils';
```

Reply:
We use this as an external utils and from the package, we did this in every test already. This would then require change in all tests, so if accepted would be in separate pull request for all tests.

Reply:
I didn't see previous E2E tests using this convention. Do you have an example where `require`  has been used in this project?

Reply:
Sure, there are many of them, one example is [here](https://github.com/Automattic/woocommerce-payments/blob/8628f9ea4c76a9f95553527e0551a24c91bb87ef/tests/e2e/specs/merchant/merchant-orders-partial-refund.spec.js#L6).